File: CSharpOOPS/guessgame/ConsoleApp1/guess/UserGuess.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace randomguess.guess
{
    class UserGuess
    {
        int number;
        public int guessNumber()
        {
            Console.WriteLine("Enter your guess");
            number=Convert.ToInt32(Console.ReadLine());
            return number;
        }
    }
}

File: OOPS/randomguess-app/src/com/ramdomguess/UserGuess.java
package com.ramdomguess;

import java.util.Scanner;

public class UserGuess {

	int userGuessGumber;
	 int guessNumber()
	 {
		 System.out.println("Enter your guess");
		 Scanner scn=new Scanner(System.in);
		 userGuessGumber=scn.nextInt();	
		 return userGuessGumber;
	 }
}
